src/fromager/commands/build.py

```diff
@@ -141,6 +142,23 @@ def _build(
     req: Requirement,
     source_download_url: str,
 ) -> pathlib.Path:
+    pbi = wkctx.package_build_info(req)
+    build_tag_from_settings = pbi.build_tag(resolved_version)
+    build_tag = build_tag_from_settings if build_tag_from_settings else (0, "")
+    build_tag_as_string = f"{build_tag[0]}{build_tag[1]}"
+
+    per_wheel_logger = logging.getLogger("")
+
+    module_name = overrides.pkgname_to_override_module(req.name)
+    log_filename = (
+        module_name + "-" + str(resolved_version) + "-" + build_tag_as_string + ".log"
```

Comment:
We could probably use a less complicated name, since it's only temporary. Something like `_current.log` even, to make it stand out.

Reply:
okay, done!